app/controllers/responses_controller.rb

```diff
@@ -7,7 +7,9 @@ class ResponsesController < ApplicationController
 
   def new
     @response = Response.new
-    @questions = Question.order("RANDOM()") + DemographicQuestion.first(1)
+    @questions = Question.order("RANDOM()") + DemographicQuestion.where(
```

Comment:
@spencerldixon I am thinking I could let all DemographicQuestions through now instead of this hardcoded line?

Reply:
Let's keep it as first(1) until we have the types and options in the seeds